tests/std/tests/XOR_swap_demo/test.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
The path of this file is unconventional. Also, do you have any benchmark result for the changes?